docker-compose-simple.yml

```diff
@@ -0,0 +1,6 @@
+version: '3'
+services:
```

Comment:
Is this file still needed?

Reply:
Yes - one of the things I wanted was a known good configuration, so you can run `bin/deploy-simple-sh my-cluster-name` to validate that 1) your authentication and region are set correctly, 2) your build or params capture ran successfully, 3) and you can deploy a working container.  This gives us a known baseline even when the yale container relationships are in a broken state.